api/lokiclient.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"log"
+	"net/http"
+	"net/url"
+)
+
+type LokiClient struct {
+	baseUrl string
+}
+
+func CreateLokiClient(baseUrl string) LokiClient {
```

Comment:
We usually use `NewClient`.

`New` hints that this will create a new structure.